File: mobile/app/src/main/java/dam/isi/frsf/utn/edu/ar/delivery/broacast_receiver/AlarmReceiver.java
package dam.isi.frsf.utn.edu.ar.delivery.broacast_receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

import dam.isi.frsf.utn.edu.ar.delivery.service.NotificationService;

public class AlarmReceiver extends BroadcastReceiver{

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.d("MIRAME", "onReceive alarma: llegamos aca");
        Intent intentNotificationService = new Intent(context, NotificationService.class);
        context.startService(intentNotificationService);
    }
}

File: mobile/app/src/main/java/dam/isi/frsf/utn/edu/ar/delivery/broacast_receiver/AlarmReceiver.kt
package dam.isi.frsf.utn.edu.ar.delivery.broacast_receiver

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

import dam.isi.frsf.utn.edu.ar.delivery.service.NotificationService

class AlarmReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        Log.d("MIRAME", "onReceive alarma: llegamos aca")
        val intentNotificationService = Intent(context, NotificationService::class.java)
        context.startService(intentNotificationService)
    }
}
